crates/oxc_linter/src/rules/jest/padding_around_test_blocks.rs

```diff
@@ -0,0 +1,246 @@
+                            }
+                        }
+                        let span_between = Span::new(span_between_start, span_between_end);
+                        let content = ctx.source_range(span_between);
+                        if content.matches('\n').count() < 2 {
+                            let spaces_after_last_line = content
+                                .rfind('\n')
+                                .map_or("", |index| content.split_at(index + 1).1);
+                            ctx.diagnostic_with_fix(
+                                padding_around_test_blocks_diagnostic(Span::new(
+                                    span_between_end,
+                                    span_between_end,
+                                )),
+                                |fixer| {
+                                    fixer.replace(
+                                        span_between,
+                                        format!("\n\n{spaces_after_last_line}"),
+                                    )
+                                },
+                            );
+                        }
+                    }
+                    prev_node = Some(node);
+                }
+                NodeType::Statement => {
+                    prev_node = Some(node);
+                }
+            }
+        }
+    }
+}
+
+#[test]
+fn test() {
+    use crate::tester::Tester;
+
+    let pass = vec![
+        "test('foo', () => {});",
+        "test('foo', () => {});\n\ntest('bar', () => {});",
+        "const thing = 123;\n\ntest('foo', () => {});",
+        "{ test('foo', () => {}); }",
+        "describe('foo', () => {\ntest('bar', () => {});\n});",
+    ];
+
+    let fail = vec![
+        "test('foo', () => {});test('bar', () => {});",
+        "test('foo', () => {});\ntest('bar', () => {});",
+        "it('foo', () => {});\nfit('bar', () => {});\ntest('baz', () => {});",
+        r"
+ describe('other bar', () => {
+     const thing = 123;
+     test('is another bar w/ test', () => {
+     });
+     // With a comment
+     it('is another bar w/ it', () => {
+     });
+     test.skip('skipping', () => {}); // Another comment
+     it.skip('skipping too', () => {});
+ });xtest('weird', () => {});
+ test
+   .skip('skippy skip', () => {});
+ xit('bar foo', () => {});
+            ",
+    ];
+
+    let fix = vec![
+        (
+            "test('foo', () => {});test('bar', () => {});",
+            "test('foo', () => {});\n\ntest('bar', () => {});",
+        ),
+        (
+            "test('foo', () => {});\ntest('bar', () => {});",
+            "test('foo', () => {});\n\ntest('bar', () => {});",
+        ),
+        (
+            "it('foo', () => {});\nfit('bar', () => {});\ntest('baz', () => {});",
+            "it('foo', () => {});\n\nfit('bar', () => {});\n\ntest('baz', () => {});",
+        ),
+        (
+            r"
+describe('other bar', () => {
```

Comment:
The top of the test is missing:

https://github.com/jest-community/eslint-plugin-jest/blob/0206a8aaa1f8bc31a5bec051cb6c51d043637566/src/rules/__tests__/padding-around-test-blocks.test.ts#L12-L26